R/R.r

```diff
@@ -9,7 +9,12 @@ R <- function(options, path = tempdir(), env_vars = NULL, ...) {
     on.exit(set_path(old))
   }
 
-  in_dir(path, system_check(r_path, options, c(r_env_vars(), env_vars), ...))
+  tmp_user_profile <- tempfile("Rprofile")
+  writeLines(text = 'options(repos=c(CRAN="http://cran.rstudio.com"))',
```

Comment:
This needs to be pulled from whatever the current option is. I think you'll need `dput()` to handle multiple repos.